metricbeat/module/windows/services/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: services
+  type: group
+  description: >
+    `services` contains the status for windows services.
+  fields:
+    - name: status
```

Comment:
Can you please update this to include all of the fields being sent. 

For an example of how to use a field formatter for the uptime field see [./metricbeat/module/system/uptime/_meta/fields.yml](https://github.com/elastic/beats/blob/65ea8cf4ff50573fc18a9f9224f6e8e37a19fc25/metricbeat/module/system/uptime/_meta/fields.yml). This will cause the uptime field to be rendered nicely in Kibana (like "2 days").